### PR TITLE
Components setting

### PR DESCRIPTION
**Pull request by Bleu-Yves-Sopoude**
In this feature we:

-Deleted all the boilerplate
-Created a new directory called components
-Implemented the design only (JSX and CSS) 
-Used functional components
-Created at least 1 child component

### CODE REVIEW

**Review by Gambit142 — COMMENTED** [review]
Hi @Bleu-Yves-Sopoude 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- The current project is that of the Math-Magician project but your README contains details of the Leaderboard project. Kindly go through the README and replace all information pertaining to the Leaderboard project with that of the Math Magician project. 👍 😎 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/App.js [hunk 1]
```diff
@@ -1,22 +1,76 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+const App = () => (
+  <div className="h-screen flex items-center justify-center bg-blue-600">
+    <div className="bg-white p-6 rounded-lg shadow-lg">
+      <div className="grid grid-cols-4 gap-0.5">
```
Comment (iambenkis) [inline src/App.js]: - [ ] Once the` Calculator` is ready, you should import it and use it in your top-level component meaning the `App.js`. Unfortunately you did not create the Calculator  component in your src folder. Kindly create it and import it in the App.js file

```jsx
import Calculator from ./components/Calculator; // Import Calculator component

function App() {
  return (
    <Calculator />
  );
}
```

- [ ] Also you should make sure the `Calculator` component has at least one child component 😄 

Example 

```jsx
function Calculator() {
  return <div>{/* Calculator app design goes here */}</div>
}
```

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Bleu-Yves-Sopoude 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- App match the given design :heavy_check_mark:  

## Required changes :recycle: :recycle: 

- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment, so I can receive the notification or use Slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Bleu-Yves-Sopoude, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Descriptive Pull Request title and summary. ✔️
-  project should have a directory called `components` ✔️
- `Calculator.js` file inside the components directory. ✔️
- `Calculator.js` component should be a **functional** component. ✔️
- `Calculator.js` component match UI design ✔️
- Good use of Git Flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**